src/OpenTelemetry.Api.ProviderBuilderExtensions/AssemblyInfo.cs

```diff
@@ -24,3 +24,19 @@
 [assembly: InternalsVisibleTo("OpenTelemetry.Exporter.Console" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("OpenTelemetry.Extensions.Hosting" + AssemblyInfo.PublicKey)]
 #endif
+
+#if SIGNED
+#pragma warning disable SA1400 // Access modifier should be declared
+file static class AssemblyInfo
```

Comment:
@sharwell I couldn't find an order for the modifiers that would make SA1400/SA1206 and IDE0036 happy. I'm sure you are aware of this but I thought I would mention it.

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3588

Fixed in 1.2.0-beta.507 and newer.

Reply:
Thanks @sharwell! Fixed it up here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4735/commits/65637b0ea3d85432503067c584fdf56399787a98